config/settings.yml

```diff
@@ -180,16 +180,11 @@ edu:
 dependents:
   prefill: true
 
-pension_burial:
-  prefill: true
-  sftp:
-    relative_path: "../VETSGOV_PENSION"
-
```

Comment:
I performed a full text search in `vets-api` for `Settings.pension_burial.*`, and these settings are no longer in use. (I was investigating this because it relied on the same API key that was updated and I wanted to confirm which APIs and teams that this change would affect.)